Zetttajs Project : Phillips Hue Bulb Integration with Mock Car Sensor Driver

anil614sagar opened the pull request:
The goal for this project is to create a simple car speed tracker system by assembling a mock car, a phillips hue LED Bulb into a Zetta app running on a PC. We will connect the app to the Internet by linking the laptop with a second Zetta server running in the cloud.

A simple use case we try to demonstrate is if a car speed goes above 100 mile/hour then family at home will be alerted by turning Phillips HUE LED bulb into red in color at home. For Example, Using this app a father knows his teenage son is driving too fast sitting at home.

AdamMagaluk commented on projects/_posts/2015-02-03-Car-Speed-Tracker-System.md:
````diff
@@ -0,0 +1,461 @@
+
+  1. From the PC command line, create the project directory.
+
+  ```bash
````
All indented code snippets need to be at least 3 spaces indented to show up.

AdamMagaluk commented:
Hi Anil, great work! There were a few formatting errors probably caused by some IDE defaults. I commented on a specific line but pertains to all indented code snippets.

If you can install jekyll and run the site locally it may help weed out some of the issues. 

Another thing you will need to update the `_data/authors.yml` with your name and photo so your post doesn't show up with Brians face.

anil614sagar commented:
Hi Adam,

Thank you for the suggestions. I have corrected the formatting errors and verified using jekyll by running site locally.

I have also updated _data/authors.yml file with name and photo.

Cheers,
Anil Sagar

AdamMagaluk commented:
Looks good.
